src/other/utility/utility-en.hbs

```diff
@@ -502,15 +504,47 @@
 		<h3 class="h5"><a href="#" class="stretched-link">[Hyperlink text]</a></h3>
 		<p>Lorem ipsum dolor sit amet, consectetur adipiscing elit.</p>
 	</div>
+	<div class="col-md-4">
+		<div class="panel panel-default position-relative">
+			<div class="panel-body">
+				<img src="https://via.placeholder.com/360x203/000000/FFFFFF.png" alt="" class="img-responsive thumbnail mrgn-bttm-sm">
+				<h3 class="h5"><a href="#" class="stretched-link">[Hyperlink text]</a></h3>
```

Comment:
Panel should not be used alone to simply created a bordered box. That is a miss use of that pattern.

```suggestion
		<section class="panel panel-default position-relative">
		<header class="panel-heading">
   <h3 class="panel-title"><a href="#" class="stretched-link">[Hyperlink text]</a></h3>
   </header>
			<div class="panel-body">
				<img src="https://via.placeholder.com/360x203/000000/FFFFFF.png" alt="" class="img-responsive thumbnail mrgn-bttm-sm">
```